easybuild/easyconfigs/r/resolos/resolos-0.3.5.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'resolos'
+version = '0.3.5'
+
+homepage = 'https://gitlab.unige.ch/resolos/resolos'
+description = """Resolos is a toolkit written in Python for maintaining reproducible environments for scientific computations.
+It's main goal is to enable researchers to easily replicate environments through space (running code on HPC environment)
+and time (environment preservation for long term archival).
+For installation and detailed usage, check out the documentation."""
+
+toolchain = {'name': 'system', 'version': ''}
```

Comment:
Resolos needs Python with some non-default modules (*e.g.* PyYAML). It is unlikely that it will work in the host system without any extra dependencies. Please pick a suitable Python version from [our repo](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/p/Python) and update the `toolchain` and `dependencies` parameters of this easyconfig accordingly.

For instance, this is how it would like for Python v3.10.4:
```
toolchain = {'name': 'GCCcore', 'version': '11.3.0'}

dependencies = {
    ('Python', '3.10.4),
    ('PyYAML', '6.0'),
}
```